esx_service/vmdk-opsd

```diff
@@ -112,32 +122,65 @@ stop() {
    if [ -n "${PID}" ]; then
       echo "Stopping $OPSD_TAG with PID=${PID}"
       kill -$SIGTERM ${PID}
+
       local COUNT=0
       while kill -0 ${PID} > /dev/null 2>&1; do
          sleep ${WAIT_KILL_SEC};
+         echo "Waiting for operations in flight to complete, please do not interrupt..."
          if [ $((COUNT++)) -gt ${NUM_RETRY} ]; then
-            echo "Sending kill -${SIGKILL} to ${PID} after kill -${SIGTERM} did not stop ${PID}"
-            kill -${SIGKILL} ${PID}
+            echo "Kill -${SIGTERM} failed to stop ${PID}"
```

Comment:
so if it fails and we back off, don't we need to restore the watchdog to oversee vmdk_ops.py daemon ?

Reply:
I did think about this. My opinion is that since user wants to stop the service, and we also prompt user "Please try to force kill the process", do we really care about the watchdog in this case? We do expect user to force kill the daemon, and if they don't, the daemon is still running in un-watched status, which I feel is acceptable.

Reply:
Makes sense, but I would recommend to add to a message "Failed to stop... keeps running with limited functionality" or doc

Reply:
I agree, but how about "Failed to stop... keeps running without watchdog"? The statement "with limited functionality" is kind of unclear and might cause ambiguity/scare to the user:)